AmericanMedicalAssociation/ama-style-guide-2
# Fixes layout for IE11

## JIRA Ticket(s)
- [EWL-5471: A1 | News article masthead is broken](https://issues.ama-assn.org/browse/EWL-5471)


## Description:
Fix masthead and layout in IE11

## To Test:
- run `gulp serve`
- visit http://localhost:3000/?p=pages-news
- observe this page matches https://americanmedicalassociation.github.io/ama-style-guide-2/?p=pages-news

## Visual Regressions
A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/bugfix/EWL-5471-news-article-masthead/html_report/index.html).


## Relevant Screenshots/GIFs:
<img width="1448" alt="screen shot 2018-10-11 at 4 57 55 pm" src="https://user-images.githubusercontent.com/2271747/46836375-d1883600-cd76-11e8-96ec-09c22f672c25.png">


## Remaining Tasks
n/a

## Additional Notes
There is a corresponding PR that is a dependency
https://github.com/AmericanMedicalAssociation/ama-d8/pull/880

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
